public_html/wp-content/themes/pattern-directory/functions.php

```diff
@@ -36,6 +37,31 @@ function setup() {
 	remove_action( 'wp_head', 'WordPressdotorg\Theme\hreflang_link_attributes' );
 }
 
+/**
+ * Determine if the current site is a Rosetta site (e.g., `es-mx.wordpress.org`).
+ *
+ * This returns `false` for `translate.wordpress.org`; it's part of the Rosetta network, but isn't a Rosetta site.
+ *
+ * @return bool
+ */
+function is_rosetta_site() : bool {
```

Comment:
I wonder if we could move these rosetta helper functions into a `mu-plugin`?... maybe in the new [`wporg-mu-plugins` repo](https://github.com/WordPress/wporg-mu-plugins). If we needed them in the pattern directory, we are bound to need them elsewhere.

Reply:
Good point, it's definitely better to put this kind of function in one place and reuse them. I'll see how to put it together.

Reply:
These functions already exist [in the global blocks,](https://github.com/WordPress/wporg-mu-plugins/blob/b1ac446e2305dbd70c1d10950f1dc3f8680461af/mu-plugins/blocks/global-header-footer/blocks.php#L453) so you should be able to use them by pulling in `use function WordPressdotorg\MU_Plugins\Global_Header_Footer\{ get_rosetta_name, is_rosetta_site }`.

A possible improvement could be to move the functions out of the blocks folder so it's clear they're not tied to the blocks.

Reply:
Thanks @ryelle! Didn't know those two functions can be pulled in that way here.

> A possible improvement (to the mu-plugins project) could be to move the functions out of the blocks folder so it's clear they're not tied to the blocks.
I'll take a look at it. 👀

Reply:
> These functions already exist [in the global blocks,](https://github.com/WordPress/wporg-mu-plugins/blob/b1ac446e2305dbd70c1d10950f1dc3f8680461af/mu-plugins/blocks/global-header-footer/blocks.php#L453) so you should be able to use them by pulling in use function WordPressdotorg\MU_Plugins\Global_Header_Footer\{ get_rosetta_name, is_rosetta_site }.

Done changing.

> A possible improvement (to the mu-plugins project) could be to move the functions out of the blocks folder so it's clear they're not tied to the blocks.
>> I'll take a look at it. 👀

I've created a new issue in wporg-mu-plugins [repo](https://github.com/WordPress/wporg-mu-plugins/issues/191) I'll keep track of it.